L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -1412,6 +1433,10 @@ void TrackletLUT::writeTable() const {
 
   out = openfile(settings_.tablePath(), name, __FILE__, __LINE__);
 
+  if (nbits_ == 0) {
+    throw cms::Exception("LogicError") << "Error in " << __FILE__ << " nbits_ == 0 ";
+  }
```

Comment:
@aryd @tomalin Can this check be moved to the beginning of `TrackletLUT::writeTable`, before `settings_.writeTable()` is checked?

That way, the exception is thrown regardless of whether `writeTable_` is set or not.